packages/clerk-js/src/core/resources/User.ts

```diff
@@ -146,6 +154,40 @@ export class User extends BaseResource implements UserResource {
     return new ExternalAccount(json, this.path() + '/external_accounts');
   };
 
+  generateTOTP = async (): Promise<TOTPResource> => {
```

Comment:
Should we align the new methods naming with the existing ones?
- generateTOTP -> `createTOTP()`
- deleteTOTP -> `disableTOTP()`

Reply:
Sure